### ISSUE TITLE
Unused ConfigMap in deploy manifest

### ISSUE THREAD

**Issue by jawnsy** [GoogleCloudPlatform/gke-autoneg-controller#96]
The deploy manifest includes a `ConfigMap` that does not appear to be used (it is not mounted to the `Deployment`). Can it be removed?

https://github.com/GoogleCloudPlatform/gke-autoneg-controller/blob/b8b9a87c65f3bfc07aebc60649b4f5c4af7d3230/deploy/autoneg.yaml#L189-L222

**Comment by rosmo** [comment]
Yeah, I don't think that is important - happy to take a patch to remove it. I think it made it's way into the Terraform example as well.
